misc/debug-adapter/modules/debug-adapter-core/src/main/java/org/ballerinalang/debugadapter/launch/ProgramLauncher.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright (c) 2019, WSO2 Inc. (http://wso2.com) All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.ballerinalang.debugadapter.launch;
+
+import org.ballerinalang.debugadapter.config.ClientConfigurationException;
+import org.ballerinalang.debugadapter.config.ClientLaunchConfigHolder;
+import org.ballerinalang.debugadapter.utils.OSUtils;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+/**
+ * Abstract implementation of Ballerina program launcher.
+ */
+public abstract class ProgramLauncher {
+
+    protected final String projectRoot;
+    protected final ClientLaunchConfigHolder configHolder;
+
+    private static final String BAL_RUN_CMD_NAME = "run";
+    private static final String BAL_TEST_CMD_NAME = "test";
```

Comment:
How about changing the name to `BAL_RUN_CMD` and `BAL_TEST_CMD` here?

Reply:
+1 changed